modules/ocf/templates/firefox/prefs.js.erb

```diff
@@ -1,3 +1,25 @@
+// This is the Debian specific preferences file for Firefox ESR
+// You can make any change in here, it is the purpose of this file.
+// You can, with this file and all files present in the
+// /etc/firefox-esr directory, override any preference you can see in
+// about:config.
+//
+// Note that pref("name", value, locked) is allowed in these
+// preferences files if you don't want users to be able to override
+// some preferences.
+
+pref("extensions.update.enabled", true);
+// Use LANG environment variable to choose locale
+pref("intl.locale.requested", "");
+
+// Default to classic view for about:newtab
+pref("browser.newtabpage.enhanced", false, sticky);
+
+// Default to no suggestions in the urlbar. This still brings a panel asking
+// the user whether they want to opt-in on first use.
+pref("browser.urlbar.suggest.searches", false);
+
+// OCF modifications made after here
 lockPref("browser.startup.homepage", "<%= @browser_homepage %>");
```

Comment:
`lockPref` is going away in a future Firefox version too, we ought to change this too